src/open_inwoner/templates/pages/profile/appointments.html

```diff
@@ -24,18 +24,12 @@ <h2 class="card__heading-2">{{ appointment.title }}</h2>
                             {% list_item text=appointment.branch.addressLine2 compact=True strong=False %}
                         {% endrender_list %}
 
-                        <span class="link link--icon link--secondary"
-                            aria-label="{% trans "Wijzig afspraak" %}"
-                            title="{% trans "Wijzig afspraak" %}">
-                            <span class="link__text">{% trans "Wijzig afspraak" %}</span>
+                        <a href="{{ booking_base_url }}{{ appointment.publicId }}" class="link link--icon link--secondary"
```

Comment:
It is a bit dirty to build URL's in the template if it is not necessary or legacy patterns.